v22.1/explain-analyze.md

```diff
@@ -76,6 +76,8 @@ KV rows read | During scans, the number of rows in the [storage layer](architect
 KV bytes read | During scans, the amount of data read from the [storage layer](architecture/storage-layer.html) during this phase of the statement.
 estimated max memory allocated  | The estimated maximum allocated memory for a statement.
 estimated max sql temp disk usage | The estimated maximum temporary disk usage for a statement.
+MVCC step count (ext/int) | The number of times that the underlying storage iterator stepped forward during the work to serve the operator's reads, including stepping over MVCC keys that could not be used in the scan.
```

Comment:
Should we link MVCC (or MVCC keys) to https://www.cockroachlabs.com/docs/stable/architecture/storage-layer.html#mvcc?